### DISCUSSION TITLE
Database won't start

### DISCUSSION THREAD

**Discussion by trickster** [neondatabase/neon#2065] [General]
I got my invite, and I created and deleted a couple of databases. Right now, I would like to create new one for experiments, but I get this error on Neon UI

```
Request failed with status 406: could not start project
```

The database won't start.

**Comment by kelvich** [comment]
Hi! We had a small incident, please try now -- your cluster should work.

**Comment by raoufchebri** [comment]
Hey @trickster, is this the database working for you now?

**Reply by trickster** [reply]
It's working now, but how does NoTls work here https://neon.tech/docs/integrations/rust/? 

It's only working with TlsConnector

```rust
let cert_path = std::env::var("HOME").unwrap() + "/.postgresql/root.crt";
    let cert = std::fs::read(cert_path)?;
    let cert = openssl::x509::X509::from_pem(&cert).unwrap();
    let mut ctx = SslConnector::builder(SslMethod::tls())?;
    ctx.set_certificate(&cert)?;

    let connector = MakeTlsConnector::new(ctx.build());

    let mut client = Client::connect(
        "postgres://conn/main",
        connector,
    )?;
```
